etc/adminhtml/system.xml

```diff
@@ -21,6 +21,10 @@
                         <field id="enabled">1</field>
                     </depends>
                 </field>
+                <field id="is_eu_cdn" translate="label" type="select" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Please let's rename this here and everywhere else to `use_eu_cdn`. Make sure to also use the right ID in the integration tests ;-)